compiler/util/llvmDebug.cpp

```diff
@@ -124,14 +125,14 @@ LLVM_DITYPE debug_data::construct_type(Type *type)
   const char* name = type->symbol->name;
   ModuleSymbol* defModule = type->symbol->getModule();
   const char* defFile = type->symbol->fname();
-  if (strstr(defFile, "/modules/")!=NULL || strcmp(defFile, "<internal>")==0) {
+  /*if (strstr(defFile, "/modules/")!=NULL || strcmp(defFile, "<internal>")==0) {
 #if HAVE_LLVM_VER >= 37
     return NULL;
 #else
     return llvm::DIType();
 #endif
 
-  }
+  }*/
```

Comment:
1) check that --llvm -g is working with 3.7 without these changes
2) migrate some of your changes to be in #if HAVE_LLVM > 33 or the reverse